### PR TITLE
Includes Markdown AST Metadata into non-code Cells

### PR DESCRIPTION
**Pull request by pastuxso**
Metadata sample output:

```json
{
  "Children": [
    {
      "Kind": "Text",
      "Text": "Paragraph 1 with a link "
    },
    {
      "Children": [
        {
          "Kind": "Text",
          "Text": "Link1"
        }
      ],
      "Destination": "https://example.com",
      "Kind": "Link",
      "Title": "Link Title 1"
    },
    {
      "Kind": "Text",
      "Text": " and a second link "
    },
    {
      "Children": [
        {
          "Kind": "Text",
          "Text": "Link2"
        }
      ],
      "Destination": "https://example2.com",
      "Kind": "Link",
      "Title": "Link Title 2"
    },
    {
      "Kind": "Text",
      "Text": ".."
    }
  ],
  "Kind": "Paragraph",
  "RawText": "Paragraph 1 with a link [Link1](https://example.com 'Link Title 1') and a second link [Link2](https://example2.com 'Link Title 2').."
}
```

### CODE REVIEW

File: internal/document/editor/cell.go [hunk 1]
```diff
@@ -144,6 +145,11 @@ func toCellsRec(
 
 		case *document.MarkdownBlock:
 			value := block.Value()
+			astNode := block.Unwrap()
+
+			metadata := &map[string]string{}
```
Comment (adambabik) [inline internal/document/editor/cell.go:150]:
```suggestion
			metadata := map[string]string{}
```

`map[K]V{}` is already a pointer (the same for `make(map[K]V)`).

Comment (pastuxso) [reply internal/document/editor/cell.go]: It was fixed, Thank you @adambabik :+1:

**Comment by sourishkrout** [comment]
I'm undecided on using the `runme.dev/` prefix for e.g. `"runme.dev/3/Kind"`. We introduce it to pass additional information between extension <> kernel that's exempt from serialization. The metadata list is getting a bit busy using the same exact mechanism. Have you tried grouping the AST information under a single property, such as `runme.dev/ast` and just nest them @pastuxso?

**Comment by sourishkrout** [comment]
@pastuxso I like it better 👍 . Scopes/namespaces it under a single property.

**Review by sourishkrout — APPROVED** [review]
👍 LGTM. I saw some minor issues inside the AST info. However, let's merge for now and circle back later.

**Pull request merged**